dev-util/webstorm/webstorm-2017.2.2.172.3757.55.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils versionator
+
+SLOT="$(get_major_version)"
+MY_PV="$(get_version_component_range 4-6)"
+MY_PN="WebStorm"
+
+DESCRIPTION="JavaScript IDE for client- and server-side development with Node.js"
+HOMEPAGE="http://www.jetbrains.com/webstorm"
+SRC_URI="http://download.jetbrains.com/${PN}/${MY_PN}-$(get_version_component_range 1-3).tar.gz"
+
+LICENSE="WebStorm WebStorm_Academic WebStorm_Classroom WebStorm_OpenSource WebStorm_personal"
```

Comment:
Hmm, isn't this more like `|| ( ... )`, i.e. any of the licenses can be chosen?

Reply:
I haven't changed this line because I don't know much about licensing.
However, it looks almost exactly like the one in the pycharm-professional ebuild: https://github.com/gentoo/gentoo/blob/a2023adbf400a1fd419bdfd7656312885dd1eb24/dev-util/pycharm-professional/pycharm-professional-2017.1.5.ebuild#L12

Reply:
WFM